### ISSUE TITLE
Add ability to change speed of song

### ISSUE THREAD

**Issue by catzs** [timusus/Shuttle#117]
I love listening to some music a bit faster than the original speed. VLC is great at this; offering the ability to speed up the song WITHOUT changing the pitch. It's also open source ;)

**Comment by timusus** [comment]
Depends on #245

**Comment by timusus** [comment]
Duplicate of #41
